config/userProfileConfig.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * This configuration file defines how user profile data should be should be deleted or masked during user deletion.
+ * - `preserveAndMask`: Fields that will be masked (not deleted).
+ * - `fieldsToRemove`: Fields that will be completely removed.
+ * - `defaultMaskedDataPlaceholder`: The default value used to mask preserved fields (e.g., "Deleted User").
+ * This is used by the PII user data deletion
+ * Developers can modify this file to change the deletion behavior:
+ * - Add/remove keys in `fieldsToRemove` to control which fields get deleted.
+ * - Add/remove keys in `preserveAndMask` to control which fields are masked instead of deleted.
+ * - Optionally define specific placeholder values in `specificMaskedValues` to set custom text instead of the default placeholder (currently commented out).
+ */
+const userProfileKeys = {
```

Comment:
@priyanka-TL - Make this default as per ELEVATE user service not sunbird. Also change the code to read from config.json . The config.json file path will come from ENV, but a default value can be give in envVariables.js